plugins/newrelic-dashboard/README.md

```diff
@@ -0,0 +1,79 @@
+# New Relic Dashboard Plugin
+
+Welcome to the new-relic-dashboard plugin!
+
+## Features
+
+- Adds New Relic Dashboard Pages Links to Overview section of the catalog
+- Shows Snapshots of dashboards in New Relic
+
+## Getting started
+
+This plugin uses the Backstage proxy to securely communicate with New Relic's APIs. We use NerdGraph (New Relic's GraphQL API)
+
+To generate a New Relic API Key , you can visit this [link](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher)
```

Comment:
You point to using a User Key, which New Relic indicates is the newer (preferred) approach. Maybe need to tweak this instruction for end-users with a new link?

Reply:
Yes , we are using the newer approach , will add some more details around it.